cli.js

```diff
@@ -65,7 +83,8 @@ dns.lookup('emoji.getdango.com', err => {
 process.stdin.on('keypress', (ch, key) => {
 	key = key || {};
 
-	if (hasAnsi(key.sequence)) {
+	// Filter out all Ansi sequences except the Up/Down keys which change the skin tone
+	if (hasAnsi(key.sequence) && key.name !== 'up' && key.name !== 'down') {
```

Comment:
These should only activate when there are search results. The user shouldn't be able to accidentally change the skin tone when they're not seeing any emoji.